functions/common/function-test-support/pom.xml

```diff
@@ -47,8 +47,12 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>
-            <version>${test-containers.version}</version>
+            <scope>compile</scope>
```

Comment:
But you said yourself in other place that this one is a default scope…

Reply:
I did not update the original changes - it was copied over.